kumascript/src/info.js

```diff
@@ -116,14 +116,18 @@ const info = {
     //   }
     //   return result;
     // }
-
-    return [];
+    return info.getPage(url, true).translations;
   },
 
-  getPage(url) {
+  getPage(url, throwIfDoesNotExist = false) {
     const result = Document.findByURL(info.cleanURL(url), { metadata: true });
     if (!result) {
-      throw new Error(`${info.getDescription(url)} does not exist`);
+      // The macros expect an empty object if the URL does not exist, so
+      // "throwIfDoesNotExist" should only be used within "info" itself.
+      if (throwIfDoesNotExist) {
```

Comment:
Is this parameter used somewhere? I can't seem to find a call site

Reply:
Yes, it's used in the `getChildren()` and `getTranslations()` calls above. I added it to `getTranslations()` just so it throws an error if its `url` parameter doesn't exist.

I would prefer to make it a keyword parameter (if was using Python), so it's clear what's going on when it's used, but in JS it felt more awkward to do:
```js
  getPage(url, { throwIfDoesNotExist = false } = {}) {
```
with the calls using:
```js
    const page = info.getPage(url, { "throwIfDoesNotExist": true } );
```
just for the two other uses in `info`, but maybe that's preferable?

Reply:
Oh that's why I misread that, I did think it was an object already. I'd use an object here too yup, to make it clear at-call-site what the boolean indicates. Unfortunately JS doesn't have keyword args so the way you described, would be the way to do it indeed.